lib/rucio/core/replica.py

```diff
@@ -39,13 +39,17 @@
 
 from __future__ import print_function
 
+import logging
 from collections import defaultdict
 from copy import deepcopy
 from curses.ascii import isprint
 from datetime import datetime, timedelta
 from json import dumps
 from re import match
 from traceback import format_exc
+import requests
```

Comment:
requests is currently not a server dependency

Reply:
whoops, my bad. I thought that was only for the clients.

Reply:
now in alphabetical order